mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -430,11 +430,14 @@ class KMeans private (
 
 /**
  * Top-level methods for calling K-means clustering.
+ * @since 0.8.0
  */
 object KMeans {
 
   // Initialization mode names
+  /** @since 0.8.0 */
```

Comment:
Yes, those are public APIs. Variables are compiled into public Java methods in Scala.